tests/test_task_engine.py

```diff
@@ -1085,9 +1159,14 @@ async def async_task():
         assert state.data.storage_key == "foo-bar"
 
     async def test_task_result_persistence_references_absolute_path(
-        self, prefect_client
+        self, enable_client_side_task_run_orchestration
     ):
-        @task(result_storage_key="test-absolute-path", persist_result=True)
+        # temporarily use a dynamic key to avoid conflicts
+        # from running this test twice in a row
```

Comment:
I tried using `tmp_path` and creating a block but it complained that the `key_path` is not absolute. I might not understand how this works but using a dynamic key avoids a conflict